src/components/user-list/user-list.jsx

```diff
@@ -1,9 +1,17 @@
+import UserPreview from '../user-preview/user-preview';
 import './user-list.css';
 
-function UserList(props) {
-    return(
-        <div className="UserList">UserList</div>
-    );
+function UserList({ users }) {
+	return (
+		<div className="UserList">
+			{users.map((user, index) => (
+				<UserPreview key={index} user={user} />
```

Comment:
ok but careful when using index as key as it may be problematic. 


> We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an in-depth explanation on the negative impacts of using an index as a key. If you choose not to assign an explicit key to list items then React will default to using indexes as keys.
[React documentation - Keys](https://reactjs.org/docs/lists-and-keys.html#keys)